src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -131,8 +133,10 @@ void AppHost::_HandleCreateWindow(const HWND hwnd, const RECT proposedRect)
 
     auto initialSize = _app.GetLaunchDimensions(dpix);
 
-    const short _currentWidth = gsl::narrow<short>(ceil(initialSize.X));
-    const short _currentHeight = gsl::narrow<short>(ceil(initialSize.Y));
+    const short _currentWidth = Utils::ClampToShortMax(

```

Comment:
If we do end up clamping the window size when we create it, what happens to the buffer? Does it immediately on creation get resized to only fill the available space?

Reply:
I investigated a bit and it looks like the buffer resizes properly. Maybe someone more familiar with this codebase could confirm, but it looks alright to me. I also tried clamping the window size to something like 300 so I could see what's going on and it looks like the buffer resizes on launch exactly as expected.